mail_activity_board/views/mail_activity_view.xml

```diff
@@ -0,0 +1,216 @@
+                            <field name="date_deadline"
+                                   attrs="{'invisible': [('activity_category', '=', 'meeting')]}"/>
+                            <field name="calendar_event_id_start" string="Start meeting"
+                                   attrs="{'invisible': [('calendar_event_id','=', False)], 'readonly': 1}"/>
+                            <field name="duration" widget="float_time"
+                                   attrs="{'invisible': ['|',('duration', '=', False),
+                                   ('calendar_event_id','=', False)], 'readonly': 1}"/>
+                            <field name="user_id"/>
+                            <field name="calendar_event_id_partner_ids" widget="many2many_tags"
+                                   attrs="{'invisible': [('calendar_event_id','=', False)], 'readonly': 1}"/>
+                        </group>
+                    </group>
+                    <field name="note" placeholder="Log a note..."/>
+
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+
+    <!-- TREE VIEW -->
+    <record id="mail_activity_view_tree" model="ir.ui.view">
+        <field name="name">boards.activities.tree</field>
+        <field name="model">mail.activity</field>
+        <field name="inherit_id" ref="mail.mail_activity_view_tree"/>
+        <field name="domain"></field>
+        <field name="arch" type="xml">
+
+            <xpath expr="//tree" position="attributes">
+
+                <attribute name="default_order"></attribute>
+
+                <attribute name="decoration-danger">(date_deadline &lt; current_date)</attribute>
+                <attribute name="decoration-warning">(date_deadline == current_date)</attribute>
+                <attribute name="decoration-success">(date_deadline &gt; current_date) </attribute>
+
+            </xpath>
+
+        </field>
+    </record>
+
+
+
+    <!-- KANBAN VIEW -->
+     <record id="mail_activity_view_kanban" model="ir.ui.view">
+         <field name="name">sdi.boards.activities.kanban</field>
+         <field name="model">mail.activity</field>
+         <field name="priority" eval="10"/>
+         <field name="context">{}</field>
+         <field name="arch" type="xml">
+            <kanban default_group_by="activity_type_id" class="_kanban_small_column o_opportunity_kanban" create="0" _order="date_deadline"
+                    group_create="false" group_delete="false" group_edit="false">
+                <field name="user_id"/>
+                <field name="res_id"/>
+                <field name="res_name"/>
+                <field name="res_model"/>
+                <field name="summary"/>
+                <field name="date_deadline"/>
+                <field name="state"/>
+                <field name="icon"/>
+                <field name="activity_type_id"/>
+                <field name="activity_category"/>
+                <templates>
+                    <t t-name="kanban-box">
```

Comment:
This whole template needs a little bit of :heart:. Check out the kanban views in crm or project modules, to inspire yourself.

Reply:
I can assure you that I have put a lot of love on this kanban card.